xarray/namedarray/utils.py

```diff
@@ -57,11 +117,19 @@ def is_duck_array(value: typing.Any) -> TypeGuard[T_DuckArray]:
     )
 
 
-def is_duck_dask_array(x: typing.Any) -> bool:
-    return is_duck_array(x) and is_dask_collection(x)
+def is_duck_dask_array(x: T_DuckArray) -> TypeGuard[DaskArray]:
+    return is_dask_collection(x)
+
+
+def is_chunked_duck_array(
+    x: T_DuckArray,
+) -> TypeGuard[_ChunkedArray[np.dtype[np.generic]]]:
```

Comment:
Using _ChunkedArray here, though I think it should be the TypeVar T_ChunkedArray.